packages/backend/src/apps/formsg/auth/verify-credentials.ts

```diff
@@ -46,12 +52,15 @@ export const verifyFormCreds = async (
     throw new Error('Form is not a storage mode form')
   }
 
-  if (!formsgSdk().crypto.valid(publicKey, secretKey)) {
+  const formSdk = getSdk(env)
```

Comment:
```suggestion
  const formsgSdk = getSdk(env)
```
can make the naming consistent here again?

Reply:
good call, thanks!